CONTRIBUTING.md

```diff
@@ -5,66 +5,78 @@ First off, thanks for taking the time to contribute! :rocket::rocket::rocket:
 The following is a set of guidelines for contributing to the **Open Space Toolkit** (OSTk) libraries, which are hosted in [Open Space Collective](https://github.com/open-space-collective) on GitHub.
 These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-## Table Of Contents
-
-- [Open Space Toolkit ▸ Contributing](#open-space-toolkit-%e2%96%b8-contributing)
-  - [Table Of Contents](#table-of-contents)
-  - [Code of Conduct](#code-of-conduct)
-  - [I Have a Question!](#i-have-a-question)
-  - [How Can I Contribute?](#how-can-i-contribute)
-    - [Reporting Bugs](#reporting-bugs)
-      - [Before Submitting A Bug Report](#before-submitting-a-bug-report)
-      - [How Do I Submit A (Good) Bug Report?](#how-do-i-submit-a-good-bug-report)
-    - [Suggesting Enhancements](#suggesting-enhancements)
-      - [How Do I Submit A (Good) Enhancement Suggestion?](#how-do-i-submit-a-good-enhancement-suggestion)
-    - [Your First Code Contribution](#your-first-code-contribution)
-    - [Pull Requests](#pull-requests)
-  - [Styleguides](#styleguides)
-    - [Git Commit Messages](#git-commit-messages)
-    - [C++ Styleguide](#c-styleguide)
-      - [File Formatting](#file-formatting)
-      - [Class Structure](#class-structure)
-      - [Method Structure](#method-structure)
-      - [Coding Rules](#coding-rules)
-        - [Acronyms](#acronyms)
-        - [Types](#types)
-      - [Syntax](#syntax)
-      - [Idioms](#idioms)
-        - [Named Constructor](#named-constructor)
-        - [PIMPL](#pimpl)
-    - [Python Styleguide](#python-styleguide)
-    - [Documentation Styleguide](#documentation-styleguide)
-  - [Additional Notes](#additional-notes)
-    - [Issue and Pull Request Labels](#issue-and-pull-request-labels)
-      - [Type of Issue and Issue State](#type-of-issue-and-issue-state)
-  - [References](#references)
-
-## Code of Conduct
+##  1. <a name='TableOfContents'></a>Table Of Contents
+
+<!-- vscode-markdown-toc -->
+* 1. [Table Of Contents](#TableOfContents)
+* 2. [Code of Conduct](#CodeofConduct)
+* 3. [I Have a Question!](#IHaveaQuestion)
+* 4. [How Can I Contribute?](#HowCanIContribute)
+	* 4.1. [Reporting Bugs](#ReportingBugs)
+		* 4.1.1. [Before Submitting A Bug Report](#BeforeSubmittingABugReport)
+		* 4.1.2. [How Do I Submit A (Good) Bug Report?](#HowDoISubmitAGoodBugReport)
+	* 4.2. [Suggesting Enhancements](#SuggestingEnhancements)
+		* 4.2.1. [How Do I Submit A (Good) Enhancement Suggestion?](#HowDoISubmitAGoodEnhancementSuggestion)
+	* 4.3. [Your First Code Contribution](#YourFirstCodeContribution)
+	* 4.4. [Pull Requests](#PullRequests)
+	* 4.5. [Git Flow](#GitFlow)
+	* 4.6. [Commit Messages](#CommitMessages)
+* 5. [Styleguides](#Styleguides)
```

Comment:
Those links in the content do not bring to the respective sections, is this expected?

Reply:
fixed